app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -369,6 +370,16 @@ class StateFragmentTest {
     Intents.release()
   }
 
+  @Test
+  fun testContentCard_forDemoExploration_withCustomOppiaTags_displaysParsedHtml() {
```

Comment:
This function should come above `tearDown` function

Reply:
done.